packages/custom-types/asset.d.ts

```diff
@@ -0,0 +1,11 @@
+import Perception from './perception'
+
+export interface Asset {
+  name: string,
+  uniqid: string,
+  position: number,
```

Comment:
Position is a string:
![image](https://user-images.githubusercontent.com/1108513/101813600-d0005700-3b14-11eb-8409-991dda08126c.png)